Sources/Tests/UnitTests/Calculus/SeriesTest.cs

```diff
@@ -52,5 +75,29 @@ public void CheckTheCorrectness(double point, int termCount, string func, int po
 
             Assert.True(error < 0.05, $"Error is: {error}");
         }
+
+        [Theory]
+        [InlineData(new double[3] { 0, 0, 0 }, 4, "sin(x) + cos(y) + tan(z)", new double[3] { 0.5, -0.5, 0.1 })]
+        [InlineData(new double[3] { 0, 0, 0 }, 4, "sqrt(x - 1) / e ^ (y - 1) + sin(z)", new double[3] { 0.1, 0.1, 0.1 })]
+        [InlineData(new double[3] { 1, 1, 1 }, 6, "(e^x)ln(1 + y)z", new double[3] { 0.5, 0.5, 0.5 })]
+        public void CheckMultivariableCorrectness(double[] point, int termCount, string func, double[] pointToCheckAt)
```

Comment:
Can we have more tests for non-zero point of differentiating? We have just one test. It would be nice to have tests for points with different coordinates, like (1, 2, 3) or (5, 0 -4).